packages/block-editor/src/components/resolution-tool/stories/index.story.js

```diff
@@ -10,67 +10,115 @@ import {
 /**
  * Internal dependencies
  */
-import ResolutionTool from '..';
+import ResolutionTool from '../';
 
-export default {
-	title: 'BlockEditor/ResolutionControl',
+const meta = {
+	title: 'BlockEditor/ResolutionTool',
 	component: ResolutionTool,
 	tags: [ 'status-private' ],
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'A control for selecting image resolution with preset size options.',
+			},
+		},
+	},
 	argTypes: {
-		panelId: { control: false },
-		onChange: { action: 'changed' },
+		value: {
+			control: 'radio',
+			options: [ 'thumbnail', 'medium', 'large', 'full' ],
+			description: 'Currently selected resolution value.',
+			table: {
+				type: { summary: 'string' },
+				defaultValue: { summary: 'thumbnail' },
+			},
```

Comment:
- The value is managed by the `useState` hook, so no control is needed.
- I think `defaultValue` is unnecessary.

```javascript
		value: {
			control: {
				type: null,
			},
			description: 'Currently selected resolution value.',
			table: {
				type: { summary: 'string' },
			},
		},
```